libbeat/tests/system/test_dashboard.py

```diff
@@ -71,14 +71,14 @@ def test_export_dashboard(self):
 
         self.test_load_dashboard()
 
-        command = self.beat_path + "/../dev-tools/cmd/dashboards/export_dashboards -kibana http://" + \
+        command = self.beat_path + "/../dev-tools/cmd/dashboards/export_dashboards.go -kibana http://" + \
             self.get_kibana_host() + ":" + self.get_kibana_port()
 
         if os.name == "nt":
-            command = self.beat_path + "\..\dev-tools\cmd\dashboards\export_dashboards -kibana http://" + \
+            command = self.beat_path + "\..\dev-tools\cmd\dashboards\export_dashboards.go -kibana http://" + \
```

Comment:
There could be just one command for all OSes if os.path.join and os.path.normpath were used to construct path part.